how-to-make-new-posts.md

```diff
@@ -0,0 +1,31 @@
+Posts Creation
```

Comment:
There are already many files in the root folder, even worse, many of them .md.
It would be better to create a new folder for readmes and add this file here.